azurerm/internal/services/network/resource_arm_web_application_firewall_policy.go

```diff
@@ -332,11 +418,102 @@ func expandArmWebApplicationFirewallPolicyPolicySettings(input []interface{}) *n
 	return &result
 }
 
+func expandArmWebApplicationFirewallPolicyManagedRulesDefinition(input []interface{}) *network.ManagedRulesDefinition {
+	if len(input) == 0 {
+		return nil
+	}
+	v := input[0].(map[string]interface{})
+
+	exclusions := v["exclusion"].([]interface{})
+	managedRuleSets := v["managed_rule_set"].([]interface{})
+
+	result := network.ManagedRulesDefinition{
```

Comment:
We could just return the struct?
```suggestion
	return &network.ManagedRulesDefinition{
```

Reply:
Done in 9fa3f6ea0.